### PR TITLE
Bugfix. NPE in TrivialPropertyAccessors rule

### LINKED ISSUE #511: NPE in TrivialPropertyAccessors rule
## Describe the bug
https://github.com/cqfn/diKTat/runs/1374670468
```
Resolving .editorconfig files for TestConfig.kt file path
Caused by: kotlin.KotlinNullPointerException
    at org.cqfn.diktat.ruleset.rules.TrivialPropertyAccessors.handleSetAccessor (TrivialPropertyAccessors.kt:59)
    at org.cqfn.diktat.ruleset.rules.TrivialPropertyAccessors.handlePropertyAccessors (TrivialPropertyAccessors.kt:52)
    at org.cqfn.diktat.ruleset.rules.TrivialPropertyAccessors.visit (TrivialPropertyAccessors.kt:44)
```

## Environment information
* diktat version: 0.1.4-SNAPSHOT

### PR DESCRIPTION
**Pull request by aktsay6**
TrivialPropertyAccessors rule throws NPE

This pull request closes #511

### CODE REVIEW

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt [hunk 1]
```diff
@@ -61,4 +61,17 @@ class TrivialPropertyAccessorsWarnTest : LintTestBase(::TrivialPropertyAccessors
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(TRIVIAL_ACCESSORS_ARE_NOT_RECOMMENDED)
+    fun `should not trigger on private setter`() {
```
Comment (petertrr) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt:67]: Please also add case with empty getter: they are useless but technically allowed in the language.
```kotlin
val x = 0
    get
```

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt [hunk 2]
```diff
@@ -74,4 +74,18 @@ class TrivialPropertyAccessorsWarnTest : LintTestBase(::TrivialPropertyAccessors
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(TRIVIAL_ACCESSORS_ARE_NOT_RECOMMENDED)
+    fun `should not trigger on getter without braces`() {
```
Comment (petertrr) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt:80]: It should or it should not?
```suggestion
    fun `should trigger on getter without braces`() {
```

Comment (aktsay6) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt:80]: Definetly should

**Review by petertrr — APPROVED** [review]
lgtm

**Pull request merged**